react-frontend/src/containers/App/index.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
@ryleyw I tend to keep imported packages together, then imported component (all together) then imported css (all together)

Reply:
Makes for less imports!